### DISCUSSION TITLE
4 HD 200's for Sale with remotes and 1 HD 100, no remote.

### DISCUSSION THREAD

**Discussion by jeffleppard1987** [OpenSageTV/Community#55] [Marketplace]
4 HD 200's for Sale with remotes and 1 HD 100, no remote.

**Comment by Sagetvmaster** [comment]
Hi.  Located in the U.S.?  What's your asking per HD200, before shipping costs.  Thanks

**Reply by jeffleppard1987** [reply]
Yes, I am in the United States. 60.00 per unit for  the HD200

**Comment by jeffleppard1987** [comment]
I can go down to 50.00 per unit, if still interested.
